vendor/bat-native-ads/src/bat/ads/internal/account/statement/statement_util.cc

```diff
@@ -27,30 +27,21 @@ base::Time GetNextPaymentDate(const TransactionList& transactions) {
 }
 
 double GetEarningsForThisMonth(const TransactionList& transactions) {
-  const base::Time& from_time = GetTimeAtBeginningOfThisMonth();
-  const base::Time& to_time = base::Time::Now();
+  const double unreconciled_earnings = GetUnreconciledEarnings(transactions);
 
-  return GetEarningsForDateRange(transactions, from_time, to_time);
-}
-
-double GetUnreconciledEarningsForPreviousMonths(
-    const TransactionList& transactions) {
-  const base::Time& from_time = base::Time();
-  const base::Time& to_time = GetTimeAtEndOfLastMonth();
+  const double reconciled_earnings_this_month =
+      GetReconciledEarningsForThisMonth(transactions);
 
-  return GetUnreconciledEarningsForDateRange(transactions, from_time, to_time);
+  return unreconciled_earnings + reconciled_earnings_this_month;
```

Comment:
Calculates earnings this month by adding up all unreconciled transactions and reconciled transactions for this month